WebJobs.Script.sln

```diff
@@ -160,6 +160,49 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "BlobTrigger-Batch", "BlobTr
 		sample\BlobTrigger-Batch\run.bat = sample\BlobTrigger-Batch\run.bat
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "EasyTables", "EasyTables", "{3EACC3F3-2747-42D6-B826-6A150F9EA92A}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "CSharp", "CSharp", "{9A295BFF-005D-456B-BBEA-6F4F389B7A66}"
+	ProjectSection(SolutionItems) = preProject
+		sample\EasyTables\CSharp\host.json = sample\EasyTables\CSharp\host.json
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "QueueToOutItem", "QueueToOutItem", "{5DF1393D-B53A-47C3-895D-A7F3071F44F1}"
```

Comment:
Need a sample in Node.js and/or one of the other languages to ensure bindings work across all languages. Also, all your samples are queue trigger based, how about an http?